### ISSUE TITLE
Noir working on MacOS 10.14

### ISSUE THREAD

**Issue by cgoguyer** [ibnishak/Timimi#34]
I've installed timini file on my mac with success, the ff extension as well, but each time I try to save my tiddly file, I get an error message:
Automatic saving not possible.
As your TW is not within the contolled directory a manual save is required
and then I get the menu to save it in the download folder !
Any idea ?

**Comment by ibnishak** [comment]
Did you try a different directory?

**Comment by cgoguyer** [comment]
Yes, i tried in home directory, document and some other folders. Same issue.

**Comment by ibnishak** [comment]
Ok. Please answer the following questions

1. Firefox version
1. Have you been using timimi before updating to Mac OS 10.14? Did it used to work? Because Mac OS 10.14 had changed their permission model.

**Comment by cgoguyer** [comment]
1) I'm using Firefox Quantum 68.2.0esr (latest release)
2) No i've just installed timimi on Mojave last week. I'm currently using TiddlyDesktop to save my wikis.

**Comment by monomycelium** [comment]
Is this issue still happening?
